package.json

```diff
@@ -61,24 +61,30 @@
     "@hmcts/frontend": "0.0.39-alpha",
     "@hmcts/media-viewer": "^1.1.12",
     "@hmcts/nodejs-healthcheck": "^1.5.1",
-    "@hmcts/rpx-xui-common-lib": "0.0.19",
+    "@hmcts/properties-volume": "^0.0.9",
+    "@hmcts/rpx-xui-common-lib": "0.0.27",
+    "@ng-idle/core": "^8.0.0-beta.4",
+    "@ng-idle/keepalive": "^8.0.0-beta.4",
     "@ngrx/effects": "^7.4.0",
     "@ngrx/router-store": "^7.4.0",
     "@ngrx/store": "^7.4.0",
     "@ngrx/store-devtools": "^7.4.0",
     "@nguniversal/express-engine": "^7.1.1",
     "@nicky-lenaers/ngx-scroll-to": "^1.1.1",
     "@types/applicationinsights-js": "^1.0.9",
+    "@types/config": "^0.0.36",
     "@types/crypto-js": "^3.1.43",
     "@types/sinon": "^7.0.13",
     "applicationinsights": "^1.5.0",
     "applicationinsights-js": "^1.0.20",
     "axios": "^0.18.0",
     "class-transformer": "^0.1.10",
     "classlist.js": "^1.1.20150312",
+    "config": "^3.2.5",
     "cookie-parser": "^1.4.4",
     "core-js": "^2.5.4",
     "crypto-js": "^3.1.9-1",
+    "dotenv-extended": "^2.7.1",
```

Comment:
if we're using this package, we can make it a bit more dev friendly by having the .env.defaults

Reply:
Nice to have sir. Lets do it in re-factor exercise